platform/email/platform-email-impl/src/test/java/org/codice/ddf/platform/email/impl/SmtpClientImplITCaseTest.java

```diff
@@ -111,9 +114,10 @@ public void testSendWithAttachments()
 
     server.stop();
 
-    assertThat(server.getReceivedEmailSize(), is(1));
-    Iterator emailIterator = server.getReceivedEmail();
-    SmtpMessage email = (SmtpMessage) emailIterator.next();
+    List<SmtpMessage> emails = server.getReceivedEmails();
+    assertThat(emails, is(not(empty())));
+    SmtpMessage email = emails.get(0);
+    assertNotNull(email);
```

Comment:
`assertNotNull("The email was null.", email)`

Reply:
fixed